unitary/examples/quantum_rpg/final_state_preparation/classical_frontier.py

```diff
@@ -84,6 +100,69 @@
     ]
 )
 
+
+def _hole(state: GameState, world):
+    raise UntimelyDeathException(
+        "You feel your entire existence dissolve into errors and static\n"
```

Comment:
DEL static
You repeat static on the next line.

Reply:
Done.